app/models/content_item.rb

```diff
@@ -46,7 +46,7 @@ def self.create_or_replace(base_path, attributes)
   field :_id, :as => :base_path, :type => String, :overwrite => true
   field :content_id, :type => String
   field :title, :type => String
-  field :description, :type => String
+  field :description, :type => Hash, :default => { "value" => nil }
```

Comment:
This sets the default to be a hash containing `{ "value" => nil }`, which is consistent with the approach we took in publishing api.

When you call `#description` on the model it "unpacks" the value and returns the nested string (or array in the case of there being content types).